spec/support/jasmine.json

```diff
@@ -0,0 +1,13 @@
+{
+  "spec_dir": "spec",
+  "spec_files": [
+    "**/*.js",
+    "**/*.coffee"
+  ],
+  "helpers": [
+    "../node_modules/babel-core/register.js",
+    "helpers/**/*.js"
+  ],
+  "stopSpecOnExpectationFailure": false,
+  "random": false
```

Comment:
@nathansobo Here's the `random` value that we could flip to `true`, and we could flip `stopSpecOnExpectationFailure` to `true` to stop immediately?